gamehype/forms.py

```diff
@@ -28,3 +36,21 @@ def validate_email(self, email):
         user = User.query.filter_by(email=email.data).first()
         if user is not None:
             raise ValidationError('Please use a different email address.')
+
+
+class AddGameForm(FlaskForm):
+    game_name = StringField(
+        'Game Title',
+        [validators.required(), validators.length(min=2, max=128)]
+        )
+    release_date = DateField('Release Date', format='%Y-%m-%d')
+    genre = SelectField(
+        'Genre(s)',
+        choices=[('Cool', 'Cool'), ('Uncool', 'Uncool')]
+        )
+    submit = SubmitField('Submit')
+
+    def validate_game_title(self, game_name):
```

Comment:
```suggestion
    def validate_game_name(self, game_name):
```

Reply:
validate_  needs to match the name of the form field

Reply:
(I did this locally and it works)

Reply:
Ah, that makes sense.